openshift/resultsdb-updater-test-template.yaml

```diff
@@ -109,6 +109,13 @@ objects:
           - name: resultsdb-updater-certs
             mountPath: /etc/resultsdb
             readOnly: true
+          resources:
+            requests:
+              cpu: 200m
+              memory: 256Mi
+            limits:
+              cpu: 300m
+              memory: 384Mi
```

Comment:
This seems like an oddly specific number. Where did you get this?

Reply:
Currently this is a wild guesstimate, which worked during testing (and follows the values seen in the waiverdb template).